Docs/Book/RDKit_Book.rst

```diff
@@ -489,6 +490,43 @@ Primitive        Property               Notes
 <-         "dative left"         extension
 =========  ====================  ===================
 
+Hs in SMARTS
+^^^^^^^^^^^^
+
+Hs in SMARTS are interpreted as hydrogen atoms if the equivalent atom expression would also be a valid SMILES; otherwise they are interpreted as a query for any atom with a single attached hydrogen.
```

Comment:
Perhaps add something like:

In general, when dealing with hydrogen atoms, the atom number primitive `#1` has consistent meaning and is the preferred method.

Reply:
Yeah, it totally makes sense to include something like this. I'll do it